utils/utils.go

```diff
@@ -258,18 +271,30 @@ func PowerOFF() {
 	if IsOpenRCBased() {
 		SH("poweroff") //nolint:errcheck
 	} else {
-		SH("shutdown") //nolint:errcheck
+		SH("shutdown now") //nolint:errcheck
 	}
 }
 
 func Version() string {
-	v, err := OSRelease("VERSION")
-	if err != nil {
-		return ""
+	// Check for K3s version
+	k3sBin := K3sBin()
+	if k3sBin != "" {
+		v, err := OSRelease("VERSION")
+		if err == nil {
+			return strings.ReplaceAll(v, "+k3s1-Kairos", "-")
```

Comment:
I think the rest of `ReplaceAll` calls were there for compatibility with older k3s and/or kairos versions. I would suggest we keep them unless we know it's safe to remove them now. Better avoid removing those in this PR since it's irrelevant.

Reply:
I couldn't find any reference to the use of `Version()`, I wonder if this was replaced completely when we moved to versioneer?

Reply:
Good point. Also the old code doesn't seem to do anything on some latest release:

```
[root@fedora ~]# cat /etc/kairos-release  | grep KAIROS_VERSION
KAIROS_VERSION="v3.2.3-v1.31.2-k3s1"
```
(from `quay.io/kairos/fedora:40-standard-amd64-generic-v3.2.3-k3sv1.31.2-k3s1` image)

Also, the old code seems as if it was trying to return the kairos version. The addition below is returning the k0s version. This is inconsistent.

We should use versioneer instead. It already understands kairos and k3s version. It might work for k0s too.

Reply:
for reference: https://github.com/kairos-io/kairos-sdk/blob/2bd0cc2843b8db1ec566924344a576579ad1fd2a/versioneer/versioneer.go#L40